spec/controllers/book_information_controller_spec.rb

```diff
@@ -0,0 +1,4 @@
+require 'rails_helper'
+
+describe BookInformationController do
```

Comment:
We are going to test Service in another PR, and we don't neer this Controller specs :wink: